### DISCUSSION TITLE
用单独的语言文件提供海外版PCL界面显示语言

### DISCUSSION THREAD

**Discussion by Crotina** [Meloong-Git/PCL#4821] [讨论]
就像mcSkin3D那些，有一个lang文件提供语言翻译，这种好处是即使自己的语言不受支持也可以自己翻译或者上网找，呃呃，可能pcl也可以这样

有预设语言，然后也可以自己加自定义的文件之类的

？？？

**Comment by Crotina** [comment]
(＠_＠)

**Comment by tangge233** [comment]
#4145 可以来加语言文件…

**Reply by Crotina** [reply]
> #4145 可以来加语言文件…

我嘞个豆(ー_ー)!!
